src/main/resources/reference.conf

```diff
@@ -156,4 +171,53 @@ cassandra-snapshot-store {
       parallelism-max = 8
     }
   }
+}
+
+# Configuration for the CassandraReadJournal
+cassandra-query-journal {
+  # Implementation class of the Cassandra ReadJournalProvider
+  class = "akka.persistence.cassandra.query.CassandraReadJournalProvider"
+  
+  # Absolute path to the write journal plugin configuration section
+  write-plugin = "cassandra-journal"
+  
+  # New events are retrieved (polled) with this interval.
+  refresh-interval = 3s
+  
+  # How many events to fetch in one query (replay) and keep buffered until they
+  # are delivered downstreams.
+  max-buffer-size = 100
+  
+  # The fetch size of the Cassandra select statement
+  fetch-size = 50
+  
+  # Read consistency level
+  read-consistency = "QUORUM"
+  
+  # Configure this to the day (yyyyMMdd) when the system was first started.
+  # When offset 0L is used it will look for events from this day and forward.
+  first-time-bucket = "20151120"
```

Comment:
Yes likely very inefficient queries, but they could potentially only be run once on startup of the journal. I agree this can be done in a separate PR if we figure out a good solution.